docs/standard/base-types/regular-expression-language-quick-reference.md

```diff
@@ -59,7 +59,7 @@ A character class matches any one of a set of characters. Character classes incl
 |`.`|Wildcard: Matches any single character except \n.<br /><br /> To match a literal period character (. or `\u002E`), you must precede it with the escape character (`\.`).|`a.e`|`"ave"` in `"nave"`<br /><br /> `"ate"` in `"water"`|
 |`\p{` *name* `}`|Matches any single character in the Unicode general category or named block specified by *name*.|`\p{Lu}`<br /><br /> `\p{IsCyrillic}`|`"C"`, `"L"` in `"City Lights"`<br /><br /> `"Д"`, `"Ж"` in `"ДЖem"`|
 |`\P{` *name* `}`|Matches any single character that is not in the Unicode general category or named block specified by *name*.|`\P{Lu}`<br /><br /> `\P{IsCyrillic}`|`"i"`, `"t"`, `"y"` in `"City"`<br /><br /> `"e"`, `"m"` in `"ДЖem"`|
-|`\w`|Matches any word character.|`\w`|`"I"`, `"D"`, `"A"`, `"1"`, `"3"` in `"ID A1.3"`|
+|`\w`|Matches any word character. This is equivalent to `[a-zA-Z_0-9]`.|`\w`|`"I"`, `"D"`, `"A"`, `"1"`, `"3"` in `"ID A1.3"`|
```

Comment:
This is only true for ECMAScript compliant behavior. Otherwise (the default?) it's the unicode categories listed in the table (see issue / the other docs site).

I have no good idea how to put this info in here, without duplicating https://docs.microsoft.com/en-us/dotnet/standard/base-types/character-classes-in-regular-expressions#word-character-w

Reply:
Hmm yes, correct. How about this
```suggestion
|`\w`|Matches any word character. For more information on the behavior of `\w`, see [Character classes: Word character](character-classes-in-regular-expressions.md#word-character-w).|`\w`|`"I"`, `"D"`, `"A"`, `"1"`, `"3"` in `"ID A1.3"`|
```

Reply:
A similar one could be added to each item of the enumeration. But in the heading of that section is already

> For more information, see Character Classes.

which links to the longer explanaition of "Character Classes".

IMO it's clear enough as it is in _main_ (but that doesn't say it shouldn't made be more clear, I just don't know how).

Reply:
Could instead clarify the intro to the table.